docs/IpInfoResultV4.md

```diff
@@ -6,6 +6,7 @@ Name | Type | Description | Notes
 **address** | **str** |  | [optional] 
 **geolocation** | [**IPLocation**](IPLocation.md) |  | [optional] 
 **asn** | [**ASN**](ASN.md) |  | [optional] 
+**datacenter** | [**DataCenterInfo**](DataCenterInfo.md) |  | [optional] 
 **data_center** | [**DataCenter**](DataCenter.md) |  | [optional] 
```

Comment:
@ilfa is there an easy way to add "Deprecated" to the description or notes here? Right now you have to click through stuff to see which is the correct one. 

```suggestion
**data_center** | [**DataCenter**](DataCenter.md) | Deprecated | [optional] 
```